src/web/components/Typography/TypographyContainer.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function connect(TypographyComponent) {
+  function Typography(props) {
+    const {
```

Comment:
you can destructure directly in the function definition:

```javascript
function Typography({ color, align, weight, transformation }) {

}
```

Reply:
(I didn't see the spread props below, sorry, this comment is not necessary

Reply:
Ok. No Problem :smile: